web-console/knikubevirt/NoStorageClassAvailable/NoStorageClassAvailable.md

```diff
@@ -0,0 +1,32 @@
+# Storage class isn’t defined
+
+Currently in our designs we have a default storage class used when a disk is added in the Create VM flow, but we do not cover the absence of a default storage class.
+This design covers the state of no storage class selected and determines how the user should be notified.
+
+During the process of creating the VM, in the Storage step, the user needs to add a disk.
+
+![Add disk action](img/Add-disk-default.png)
+
+Clicking the ‘Add disk’ button will open a modal, where the user is asked to fill in the disk’s details.
+
+We want to allow the user to set the Storage class within the wizard, with no need to be taken out of it.
```

Comment:
We should remove this line now that we can't let them create a SC in the flow.

Reply:
Curious to hear more about the spacing between the Disks table and Boot source. Is there a reason we have spread them far apart on the screen?

Reply:
Hi, are there specific guidelines for this kind of spacing? Thanks.

Reply:
We have some general spacing guidelines - have you seen these yet? https://www.patternfly.org/v4/design-guidelines/styles/spacers I will also look into if there are spacers documented for forms.

Reply:
Thanks, I updated the files with 48px spacing.